teamB/sangwoo/app.js

```diff
@@ -49,6 +49,137 @@ app.post("/user/signup", async (req, res) => {
     res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts", async (req, res) => {
+    const { title, content, user_id } = req.body;
+
+    await myDataSource.query(
+        ` 
+        INSERT INTO posts(
+            title,
+            content,
+            user_id
+        ) VALUES (?,?,?);
+        `,
+        [title, content, user_id]
+    );
+    res.status(201).json({ message: "postCreated" });
+});
+
+
+app.get("/posts", async (req, res) => {
+    await myDataSource.manager.query(
+        `SELECT 
+            u.id AS userId,
+            u.profile_Image AS userProfileImage,
+            p.id AS postingId,
+            p.title AS postingTitle,
+            p.content AS postingContent
+        FROM users u
+        INNER JOIN posts p ON
+        u.id = p.user_id
```

Comment:
- Alias를 사용하실 때에는 통일해서 사용해주세요.
- JOIN문은 한 줄로 작성해주세요~
```suggestion
        INNER JOIN posts AS p ON u.id = p.user_id
```